python/src/scipp/compat/mantid.py

```diff
@@ -378,8 +419,17 @@ def _convert_MatrixWorkspace_info(ws):
         },
         "masks": {},
         "attrs": {
-            "run": make_run(ws),
-            "sample": make_sample(ws)
+            "run":
+            make_run(ws),
+            "sample":
+            make_sample(ws),
+            "detector_rotation":
+            rot,
+            "detector_shape":
+            shp,
+            "instrument_name":
```

Comment:
Convention is to use hyphen `-`, not `_`.

I am not sure about the `detector` prefix. We don't have it for `position`. Furthermore, the dimension is named `spectrum`, so having `detector` here may be confusing, in particular since it is *not* the detector position if grouping is present? I see that "rotation" and "shape" may sound less clear though, so that is a tradeoff.